--- /opt/eda/openram-smoke/sky130_sram_1rw_24x128_vdd_via_column_mux_candidate/sky130_sram_1rw_24x128_TT_1p8V_25C.lib	2026-08-20 22:00:30.868619656 +0700
+++ /mnt/d/TKVM_ATTT_2026_test/asic/../report/a2_macro_link_smoke/run.rjfysF/derived_liberty/sky130_sram_1rw_24x128_TT_1p8V_25C.physical.lib	2026-08-21 10:52:47.369447900 +0700
@@ -58,8 +58,8 @@
     type (data){
     base_type : array;
     data_type : bit;
-    bit_width : 24;
-    bit_from : 23;
+    bit_width : 25;
+    bit_from : 24;
     bit_to : 0;
     }
 
@@ -113,6 +113,10 @@
             address : addr0; 
             clocked_on  : clk0; 
         }
+        pin(din0[24]){
+        direction : input;
+        capacitance : 0.0;
+        }
         pin(din0[23:0]){
         timing(){ 
             timing_type : setup_rising; 
@@ -152,6 +156,9 @@
         memory_read(){ 
             address : addr0; 
         }
+        pin(dout0[24]){
+        direction : output;
+        }
         pin(dout0[23:0]){
         timing(){ 
             timing_sense : non_unate; 
@@ -181,6 +188,10 @@
         }
     }
 
+    pin(spare_wen0){
+        direction : input;
+        capacitance : 0.0;
+    }
     bus(addr0){
         bus_type  : addr; 
         direction  : input; 
